crates/bevy_render/src/mesh/shape/cone.rs

```diff
@@ -0,0 +1,137 @@
+use crate::mesh::{Indices, Mesh};
+use wgpu::PrimitiveTopology;
+
+/// A cone which stands on the XZ plane
```

Comment:
It would be nice to comment on the tip and face geometry (especially the UVs).

Reply:
Assuming this is resolved via your commit, let me know if not.

Reply:
Nope, I'd like much more detail, ala what HackerFoo was requesting below.

Reply:
I've been thinking a bit about how I would go about making some diagrams or similar, but I feel making it in an ASCII editor will be messy.

I see there is some precedent for having images in docs: https://github.com/bevyengine/bevy/blob/main/assets/docs/Mesh.png
But that is only linked to and not shown directly via `rustdoc`.

Should I just create an image?